client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -54,18 +56,20 @@ const PlaceholderRow = ({
   return quantity === null ? null : (
     <tr>
       <PlaceholderCell
-        colSpan={4 + extraColumnOffset}
+        colSpan={5 + extraColumnOffset}
         sx={{ color: 'secondary.main' }}
       >
         {t('label.placeholder')}
       </PlaceholderCell>
-      <PlaceholderCell
-        style={{ textAlign: 'right', paddingRight: '14px' }}
-        colSpan={2}
-      >
+      <PlaceholderCell style={{ textAlign: 'right', paddingRight: '14px' }}>
```

Comment:
bumping the above placeholder cell one col wider, rather than having this one (pack size) span 2 cols